charts/descheduler/templates/NOTES.txt

```diff
@@ -10,3 +10,13 @@ WARNING: You enabled DryRun mode, you can't use Leader Election.
 {{- end}}
 {{- end}}
 {{- end}}
+{{- if .Values.deschedulerPolicy }}
+A default DeschedulerPolicy has been applied for you. You can view the policy with:
```

Comment:
It's not actually default, because you can control the definition via the values file, right?